### PR TITLE
Add authentication module

### PR DESCRIPTION
**Pull request by masa-orca**
##### SUMMARY

- Add authentication module
- Add tests for authentication module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

zabbix_authentication
##### ADDITIONAL INFORMATION

- This module for Zabbix 5.4 and higher.
- Therefore, I tested sanity check and playbook on local enviroment.
- In now, This PR cannot pass checks because Some documents which belong to other modules cannot pass sanity checks. Please see [this issue](https://github.com/ansible-collections/news-for-maintainers/issues/9)

```paste below

```

### CODE REVIEW

**Review by D3DeFi — APPROVED** [review]
This is some quality code :) thank you for the contribution! :rocket: 

Working on fixes to sanity tests in #649

File: plugins/modules/zabbix_authentication.py [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+
+description:
+   - This module allows you to modify Zabbix authentication setting.
+
+author:
+    - ONODERA Masaru(@masa-orca)
+
+requirements:
+    - "zabbix-api >= 0.5.4"
+
```
Comment (D3DeFi) [inline plugins/modules/zabbix_authentication.py:25]: can you add this also please?

```suggestion
version_added: 1.6.0
```

**Comment by D3DeFi** [comment]
And please rebase your PR on the latest changes from `main` branch

**Comment by masa-orca** [comment]
Sure. I rebase this branch.

File: plugins/modules/zabbix_authentication.py (hunk 1 above)
Comment (masa-orca) [reply plugins/modules/zabbix_authentication.py:27]: Thanks! I add it.

**Comment by D3DeFi** [comment]
<3

**Pull request merged**